packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/services-error-modal.jsx

```diff
@@ -1,37 +1,53 @@
 import React         from 'react';
 import PropTypes     from 'prop-types';
+import { Dialog }    from 'deriv-components';
 import { localize }  from 'App/i18n';
-import FullPageModal from 'App/Components/Elements/FullPageModal/full-page-modal.jsx';
 import { title }     from './constants';
+import { connect }   from 'Stores/connect';
 
 const ServicesErrorModal = ({
+    disableApp,
+    enableApp,
+    is_loading,
     is_visible,
     onConfirm,
-    services_error,
+    services_error
 }) => {
     const { code, message } = services_error;
 
     if (!code || !message) return null;
 
     return (
-        <FullPageModal
+        <Dialog
             title={title[services_error.type]}
             confirm_button_text={localize('OK')}
             onConfirm={onConfirm}
             // TODO: handle onCancel
             // cancel_button_text={cancel_button_text}
             // onCancel={onCancel}
+            disableApp={disableApp}
+            enableApp={enableApp}
+            is_loading={is_loading}
             is_visible={is_visible}
         >
             {message}
-        </FullPageModal>
+        </Dialog>
     );
 };
 
 ServicesErrorModal.propTypes = {
+    disableApp    : PropTypes.bool,
+    enableApp     : PropTypes.bool,
+    is_loading    : PropTypes.bool,
     is_visible    : PropTypes.bool,
     onConfirm     : PropTypes.func,
     services_error: PropTypes.object,
 };
 
-export default ServicesErrorModal;
+export default connect(
+    ({ ui }) => ({
```

Comment:
why do you need to connect to the mobx store here?

Reply:
@kelly-binary i need to pass `disableApp` and `enableApp` to modal.  modal used to be conected directly to stores.
though later with core we will handle it there.